sunpy/net/vso/vso.py

```diff
@@ -168,6 +169,22 @@ def iter_errors(response):
         if not hasattr(prov_item, 'record') or not prov_item.record:
             yield prov_item
 
+def check_connection(url):
+    try:
+        code = urllib2.urlopen(url).getcode()
+    except Exception, e:
+        raise Exception(e)
+    
+def fallback_test(api,url,port):
+    if api is None:
+        if url is None:
+            url = DEFAULT_URL
+        if port is None:
+            port = DEFAULT_PORT
+        check_connection(url)
```

Comment:
I think each URL has a port, but I don't know what they are other than the one in that issue. The transport just needs to be HTTP/HTTPS.